browser_search_plugin.php

```diff
@@ -34,33 +34,32 @@
 require_api( 'config_api.php' );
 require_api( 'gpc_api.php' );
 
-$f_type = gpc_get_string( 'type', 'text' );
+$f_type = strtolower( gpc_get_string( 'type', 'text' ) );
 
-header( 'Content-Type: application/opensearchdescription+xml' );
-?>
-<OpenSearchDescription xmlns="http://a9.com/-/spec/opensearch/1.1/"
-                       xmlns:moz="http://www.mozilla.org/2006/browser/search/">
-<?php
 $t_path = config_get_global( 'path' );
+$t_title = config_get_global( 'search_title' );
```

Comment:
I wouldn't use config_get_global() to retrieve search title.  window_title can be specified in the db, the same should be for search title.